components/dashboard/src/settings/Preferences.tsx

```diff
@@ -53,7 +53,12 @@ export default function Preferences() {
                 We're deprecating the Theia editor. You can still switch back to Theia for the next few weeks but the preference will be removed by the end of August 2021.
             </AlertBox>
             <div className="mt-4 space-x-4 flex">
-                <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultIde === 'code'} onClick={() => actuallySetDefaultIde('code')}>
+            <SelectableCard className="w-36 h-40" title="VS Code" selected={defaultIde === 'code'} onClick={() => actuallySetDefaultIde('code')}>
+                    <div className="flex-grow flex justify-center items-center">
+                        <img className="w-16 filter-grayscale" src={vscode}/>
+                    </div>
+                </SelectableCard>
+                <SelectableCard className="w-36 h-40" title="VS Code Insiders" selected={defaultIde === 'code-insiders'} onClick={() => actuallySetDefaultIde('code-insiders')}>
                     <div className="flex-grow flex justify-center items-center">
                         <img className="w-16 filter-grayscale" src={vscode}/>
```

Comment:
**suggestion**: What do you think of using the actual VS Code Insider logo here which is slightly different? This could help regular VS Code users identify faster the difference between these two options.

Reply:
I am not such a big friend of the name "Insiders", because it's the latest port of our changes to the latest release of VS Code rather than the official VS Code Insiders (which is ahead of the current VS Code release). Do you think that matters or is that nitpicking?

Reply:
Oh, wait. Maybe I've missed the context here. I thought this was about having the option to switch to the official VS Code Insiders as the editor option. If this is about our own development on top of VS Code I'd vote for definitely removing the term _Insiders_ altogether as it could be misleading.

Using a more user frienly term like _Latest_ or _Unstable_ sound more appropriate here, right?

Reply:
How about choosing between "VS Code stable" between "VS Code latest"?
Who would choose an unstable IDE?

Reply:
Using the _Latest_ label sounds better here. I think we don't have to use any label for the  _Stable_ option within the product.